collect_app/src/main/java/org/odk/collect/android/activities/AppListActivity.java

```diff
@@ -193,6 +195,10 @@ private void setupDrawerItems() {
             @Override
             public View getView(int position, View convertView, ViewGroup parent) {
                 TextView textView = (TextView) super.getView(position, convertView, parent);
+                Log.e("selected ",mSelectedSortingOrder+"");
+                if(position == mSelectedSortingOrder) {
+                    textView.setBackgroundColor(Color.parseColor("#2196F3"));
```

Comment:
Please move this value to [colors.xml](https://github.com/opendatakit/collect/blob/master/collect_app/src/main/res/values/colors.xml)